### ISSUE TITLE
Can't declare remaps in IfWinActive

### ISSUE THREAD

**Issue by Magikh42** [phil294/AHK_X11#96]
On AutoHotkey you can declare remap in IfWinActive and the remap will occur only on windows that match the condition

Example: Remapping LAlt to F11 in World of Warcraft

AutoHotkey:
``` autohotkey
IfWinActive, World of Warcraft
{
*~LAlt::
	send, {F12 down}
	KeyWait, LAlt
	send, {F12 up}
	return
}

```
ahk_x11:
``` autohotkey
*~LAlt::
	IfWinActive, World of Warcraft
	{
		send, {F12 down}
		KeyWait, LAlt
		send, {F12 up}
		return
	}

```

That means that you have to repeat the condition for each keybind related to the window
IfWinActive is a useful feature for example to avoid game remaps to aplpy when "alt-tabbed" on an other window

**Comment by phil294** [comment]
> On AutoHotkey you can declare remap in IfWinActive and the remap will occur only on windows that match the condition

Nope. You use `#IfWinActive` for that - see #97 (well, unless this was changed in ahk v2 which I have less experience with)

You're correct that you have to repeat it for every keybind. The only other alternative would be to listen for window change events and dynamicallty remove and recreate hotkeys using `Hotkey, ...`, probably rather complicated.

Having `#IfWinActive` on here would be very helpful, I agree. It's on the official to-do list ("*Click here* to see which commands are implemented and which are missing") in the Readme, but I haven't notably worked on this project since 2023 sooooo
